assets/src/styles/blocks.scss

```diff
@@ -49,9 +49,5 @@
   margin: 0 !important;
 }
 
-.is-style-roboto-font-family {
```

Comment:
We could also leave these styles for now to unblock merging this PR, and remove them only once we have updated the existing patterns 🤔